doc/generic/pgf/pgfmanual-en-pgfsys-commands.tex

```diff
@@ -922,10 +922,14 @@ \subsection{Transparency System Commands}
     transformation should not apply to the following graphics, however.
 \end{command}
 
+\pagebreak[2]
```

Comment:
Is the `\pagebreak` really necessary? If yes, we'll hopefully remember it when something changes that makes it obsolete or or worse.

Reply:
`\pagebreak[2]` expands to something like `\vpenalty150` or so, i.e. it doesn't force a pagebreak here, but prefers it slightly.

Reply:
This `\pagebreak[2]` changes
<img width="773" alt="image" src="https://github.com/pgf-tikz/pgf/assets/6376638/43e877bb-1157-4e30-81fb-0da0ec13b6d4">
to
<img width="773" alt="image" src="https://github.com/pgf-tikz/pgf/assets/6376638/85d4c943-b21b-4197-86f0-f2209ddbd800">

As `pgfmanual-en-pgfsys-commands.tex` is relatively stable, I think we can live with this amount of manual adjustment.